invokeai/frontend/web/src/features/gallery/components/GallerySettingsPopover.tsx

```diff
@@ -69,7 +75,14 @@ const GallerySettingsPopover = () => {
             dispatch(shouldAutoSwitchChanged(e.target.checked))
           }
         />
-        <BoardAutoAddSelect />
+        <IAISimpleCheckbox
+          label={t('gallery.autoAssignBoardOnClick')}
+          isChecked={autoAssignBoardOnClick}
+          onChange={(e: ChangeEvent<HTMLInputElement>) =>
+            dispatch(autoAssignBoardOnClickChanged(e.target.checked))
+          }
+        />
+        {!autoAssignBoardOnClick && <BoardAutoAddSelect />}
```

Comment:
Rendering this component conditionally causes the popover to expand as the setting is toggled.

I'd prefer to always render `<BoardAutoAddSelect />`, and disable it when `autoAssignBoardOnClick` is enabled.